docs/advanced-guides/firehose.mdx

```diff
@@ -71,3 +70,29 @@ limited concurrency based on who the event is for.
 Once we have a scheduler, we call into `HandleRepoStream` which does the actual
 decoding of the data coming over the websocket and calls into the event handler
 we wrote.
+
+If you want a more simplified firehose where events are JSON, you can connect
+to the [JetStream](https://jazco.dev/2024/09/24/jetstream/) websocket:
+<Tabs>
+  <TabItem value="python" label="Python">
+    ```python
+    import asyncio
+    import websockets
+
+    uri = "wss://jetstream2.us-east.bsky.network/subscribe?wantedCollections=app.bsky.feed.post"
+
+    async def listen_to_websocket():
+      async with websockets.connect(uri) as websocket:
+        while True:
+          try:
+            message = await websocket.recv()
```

Comment:
maybe do something with the message here, like print it to stdout?

Reply:
Great point! Added a print and tested. Thanks~!:)